src/kvp/mod.rs

```diff
@@ -34,8 +34,12 @@ where
     InvalidKey { source: KeyError, key: String },
 
     /// Indicates that the value failed to parse.
-    #[snafu(display("failed to parse value {value:?} of key/value pair"))]
-    InvalidValue { source: E, value: String },
+    #[snafu(display("failed to parse value [{value:?}] of key [{key:?}] "))]
```

Comment:
Thanks. There was a time when we used to put "[" and "]" around interpolated values to make it clear when they are empty. Maybe things have changed and I missed something.

But in your code, if value is empty, the actual message will be:

```
failed to parse value  of key "example_key"
```

Reply:
In my example, the empty value would render an error like so:

```
failed to parse value "" of key "example_key"
```

Because the `:?` interpolation for string wraps the text in double quotes.

It isn't obvious, so I'm happy if we have a code standard to wrap values (eg: if the struct member was a number, there would be no quotes). So that should perhaps go into the style guide?

Happy to split this out into a new issue in the documentation repository to get this one moving.